src/theme-context-provider.tsx

```diff
@@ -161,6 +164,8 @@ const ThemeContextProvider: React.FC<Props> = ({theme, children, providerId}) =>
         };
     }, [theme, isOsDarkModeEnabled]);
 
+    const themeStyle = `:root {${assignInlineVars(vars, {colors: contextTheme.colors})}}`;
```

Comment:
is this needed in snapshot tests? there is a hash in this styles that will change in each css change? maybe not needed for tests snapshots?

Reply:
maybe we could start thinking if those test make sense today or we should migrate them to screenshot tests

Reply:
I've finally excluded this from tests, as Pedro suggested